docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -54,12 +54,12 @@ POST /transactions/
 
    Push a new transaction.
 
-   Note: The posted transaction should be valid `transaction
+   Note: The posted transaction should be a valid and signed `transaction
    <https://bigchaindb.readthedocs.io/en/latest/data-models/transaction-model.html>`_.
    The steps to build a valid transaction are beyond the scope of this page.
    One would normally use a driver such as the `BigchainDB Python Driver
    <https://docs.bigchaindb.com/projects/py-driver/en/latest/index.html>`_ to
-   build a valid transaction.
+   build a valid transaction for a public/private keypair.
```

Comment:
Not every transaction will have just one associated public/private keypair. For example, a 2-of-2 threshold condition would have two associated keypairs. Maybe just leave this line the way it was.